src/python/pants/goal/builtins.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+from pants.build_graph.build_configuration import BuildConfiguration
+from pants.goal import help
+from pants.goal.builtin_goal import BuiltinGoal
+
+
+def register_builtin_goals(build_configuration: BuildConfiguration.Builder) -> None:
+    build_configuration.register_subsystems("pants.goal", builtin_goals())
+
+
+def builtin_goals() -> tuple[type[BuiltinGoal], ...]:
```

Comment:
Maybe inline this? Imo the code is already pretty simple, and then you don't have extra noise like this type hint.

Reply:
It was, but it is used in arg splitter test atm.